tensorflow/contrib/libsvm/python/ops/libsvm_ops.py

```diff
@@ -22,12 +22,17 @@
 from tensorflow.python.framework import ops
 from tensorflow.python.framework import sparse_tensor
 from tensorflow.python.platform import resource_loader
+from tensorflow.python.util.deprecation import deprecated
 
 
 _libsvm_ops_so = loader.load_op_library(
     resource_loader.get_path_to_datafile("_libsvm_ops.so"))
 
-
+@deprecated(
+    '2019-03-27',
```

Comment:
I think we don't use the date argument.

Reply:
The date argument changes the warning generated, I would omit it, it's generally not accurate.